README.rst

```diff
@@ -2,3 +2,33 @@ casepropods.dummy
 ==================
 
 Example casepro data pod returning statically configured data.
+
+Install
+~~~~~~~
+
+.. code-block::
+
+  $ pip install casepropods.dummy
+
+Usage
+~~~~~
+
+In casepro's settings file, add the pod as an installed app, and configure the pods that you would like to show in the UI:
+
+.. code-block:: python
+
+  INSTALLED_APPS += ('casepropods.dummy.plugin.DummyPodPlugin',)
+
+  PODS = [{
+      'label': 'dummy_pod',
```

Comment:
Do you think it's clear enough that this label has to match the one on the `DummyPodPlugin` class, or should we explain that somehow? Maybe in a comment?